CoreZipCode/Interfaces/ApiHandler.cs

```diff
@@ -5,14 +5,32 @@
 
 namespace CoreZipCode.Interfaces
 {
+    /// <summary>
+    /// ApiHandler class
+    /// </summary>
     public class ApiHandler
     {
+        /// <summary>
+        /// ApiHandler Constructor without parameter.
+        /// </summary>
         public ApiHandler() => Request = new HttpClient();
 
+        /// <summary>
+        /// ApiHandler Constructor with parameter: request.
+        /// </summary>
+        /// <param name="request"></param>
```

Comment:
Why not explain what this parameter is? it's the HttpClient class to handle with connection to API Servers.
Like this: ` /// <param name="request">HttpClient class param to handle with API Servers Connections.</param>` but improve this text.